src/haddock/libs/libalign.py

```diff
@@ -460,8 +468,10 @@ def load_coords(
                         #     " was not matched!"
                         #     )
                         continue
-                # identifier = f"{chain}.{resnum}.{atom_name}"
-                identifier = (chain, resnum, atom_name)
+                if add_resname is True:
+                    identifier = (chain, resnum, atom_name, resname)
+                else:
+                    identifier = (chain, resnum, atom_name)
```

Comment:
Why this `add_resname`? isn't it simpler to just change the identifer to always contain the resname?

Reply:
that would not be optimal for the CAPRI module, where we may want to calculate CAPRI quantities irrespectively of the resname (if the alignment is satisfied of course)